rpc_util.go

```diff
@@ -522,6 +522,42 @@ func (o MaxRetryRPCBufferSizeCallOption) before(c *callInfo) error {
 }
 func (o MaxRetryRPCBufferSizeCallOption) after(c *callInfo, attempt *csAttempt) {}
 
+// PerRPCUnaryClientInterceptor returns a CallOption that specifies the interceptor
```

Comment:
Nit: s/specifies the/specifies an/

Given that this is just one of multiple interceptors that can specified through this call option, it would be nicer to say "an interceptor" as compared to "the interceptor".

This comment applies to all such occurrences in this file.